CMakeLists.txt

```diff
@@ -223,7 +223,7 @@ endif()
 # Depend on Composable Kernels
 option(MIOPEN_USE_COMPOSABLEKERNEL "Enable MIOpen to use composable kernels for various operations" On)
 if(MIOPEN_USE_COMPOSABLEKERNEL)
-find_package(composable_kernel 1.0.0 COMPONENTS device_operations host_tensor)
+find_package(composable_kernel 1.0.0 COMPONENTS device_operations utility)
```

Comment:
@zhanglx13 This `utility` is what I was mentioning about :)
Please try to build based on the latest docker image `rocm/miopen:ci_rocm530_ubuntu2004`